jcommune-view/jcommune-web-view/src/main/webapp/resources/javascript/app/topicDraft.js

```diff
@@ -0,0 +1,534 @@
+        this._element = $('' +
+            '<div class="alert alert-error">' +
+            '<span class="close">&times;</span>' +
+            '<span id="alertMessagePopup"></span>' +
+            '</div>');
+        this._message = $('#alertMessagePopup', this._element);
+
+        $('.close', this._element).on('click', this.hide.bind(this));
+
+        this.hide();
+    }
+
+    AlertMessagePopup.prototype = {
+        show: function (message) {
+            this._message.text(message);
+            this._element.show();
+        },
+
+        hide: function () {
+            this._element.hide();
+        },
+
+        getElement: function () {
+            return this._element;
+        }
+    };
+
+    /**
+     * Class for work with current draft topic
+     *
+     * @constructor
+     */
+    function TopicDraft(api, counter, popup) {
+
+        var self = this;
+
+        this._api = api;
+        this._counter = counter;
+        this._popup = popup;
+
+        this._title = $('#subject');
+        this._content = $('#postBody');
+        this._pollTitle = $('#pollTitle');
+        this._pollItemsValue = $('#pollItems');
+
+        var defaultError = $("<span class='help-inline focusToError'></span>");
+
+        /*
+         * Construct elements for displaying errors. If on the current page already
+         * there are such elements generated on backend, we use them.
+         */
+        this._errors = {
+            title: {
+                group: this._title.parents('.control-group'),
+                error: (function() {
+                    var error = $('#topic\\.title\\.errors');
+
+                    return error.length ? error
+                                        : defaultError.clone().insertAfter(self._title)
+                })()
+            },
+            content: {
+                group: this._content.parents('.control-group'),
+                error: (function() {
+                    var error = $('#bodyText\\.errors');
+
+                    return error.length ? error
+                                        : defaultError.clone().insertAfter(self._content);
+                })()
+            },
+            pollTitle: {
+                group: this._pollTitle.parents('.control-group'),
+                error: (function() {
+                    var error = $('#topic\\.poll\\.title\\.errors');
+
+                    return error.length ? error
+                                        : defaultError.clone().insertAfter(self._pollTitle);
+                })()
+            },
+            pollItemsValue: {
+                group: this._pollItemsValue.parents('.control-group'),
+                error: (function() {
+                    var error = $('#topic\\.poll\\.pollItems');
+
+                    return error.length ? error
+                                        : defaultError.clone().insertAfter(self._pollItemsValue)
+                })()
+            }
+        };
+
+        this._wasChanged = false;
```

Comment:
Could you please add docs here? Also, we need to put all the fields in one place (at the top or at the bottom).